pkg/cloud/vsphere/actuators/cluster/actuator.go

```diff
@@ -168,15 +181,18 @@ func (a *Actuator) reconcilePKI(ctx *context.ClusterContext) error {
 }
 
 func (a *Actuator) reconcileReadyState(ctx *context.ClusterContext) error {
-	online, controlPlaneEndpoint, err := kubeclient.GetControlPlaneStatus(ctx)
+
+	// No need to do anything if the cluster is already...ready.
+	if ok, _ := strconv.ParseBool(ctx.Cluster.Annotations[constants.ReadyAnnotationLabel]); ok {
```

Comment:
For a follow-up, can this just be an existence check?

Reply:
Or is the way we check ready using this annotation already defined?

Reply:
It can an existence check. I wasn't sure what the consensus preference was on annotations. I've seen both `true` used as well as nothing at all.